src/logger.rs

```diff
@@ -38,41 +36,74 @@ impl Default for LoggerConfig {
 /// initialise your own.
 ///
 /// Configuration of the logger can also be controlled via environment variables:
-///
+/// * AMETHYST_LOG_DISABLE_STDOUT - if set, disables logging to the terminal
 /// * AMETHYST_LOG_DISABLE_COLORS - if set, disables colors for the log output
 /// * AMETHYST_LOG_LEVEL_FILTER - sets the log level
-///
+/// * AMETHYST_LOG_FILE_PATH - if set, enables logging to the file at the path
 pub fn start_logger(mut config: LoggerConfig) {
-    if let Ok(_) = env::var("AMETHYST_LOG_DISABLE_COLORS") {
+    env_var_override(&mut config);
+    let mut dispatch = basic_dispatch(config.level_filter);
+
+    if config.use_stdout {
+        dispatch = if config.use_colors {
+            dispatch.chain(colored_stdout())
+        } else {
+            dispatch.chain(io::stdout())
+        }
+    }
+
+    if let Some(path) = config.log_file {
+        match fern::log_file(path) {
+            Ok(log_file) => dispatch = dispatch.chain(log_file),
+            Err(_) => eprintln!("Unable to access the log file, as such it will not be used"),
+        }
+    }
+
+    dispatch.apply().unwrap_or_else(|_| {
+        debug!("Global logger already set, default Amethyst logger will not be used")
+    });
+}
+
+fn env_var_override(config: &mut LoggerConfig) {
+    if env::var("AMETHYST_LOG_DISABLE_STDOUT").is_ok() {
```

Comment:
I think this should better use the positive form and check if the value is `0` / `1` and override respectively.